src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc

```diff
@@ -744,234 +744,80 @@ Defines the display element from the selected report the page block will use whe
 
 |===
 
-[#page-block-geometry]
-== Geometry
+[#page-block-tabs]
+== Tabs
 
-The *geometry page block* allows you to show a map with markers, lines and polygons from different geometry values of different record sources.
+The *tabs* page block allows the grouping of different page blocks together, decluttering, and making it easier to navigate a page.
 
-image:compose-configuration/page-block-reference/config-geometry.png[...,49%,role="preview data-zoomable"]
-image:compose-configuration/page-block-reference/added-geometry.png[...,49%,role="preview data-zoomable"]
+=== Configuring Tabs
```

Comment:
This doesn't belong here.
Page block reference provides a glance over the available page blocks.
If we want to go into details how to use something, how to add this thing, etc. move it somewhere else in the low-code config sections.

Since this mostly talks about how to add the page block, I'd just remove it since this is discussed elsewhere and not that complicated to re-explain it here.